package.json

```diff
@@ -6,6 +6,13 @@
     "@capacitor/android": "2.3.0",
     "@capacitor/core": "2.3.0",
     "@capacitor/ios": "2.2.0",
+    "@fortawesome/fontawesome-pro": "^5.14.0",
```

Comment:
Die Dächelchen müssen noch weg, damit wir keine Probleme mit der Lizenz bekommen. Und die Lizenzbedingungen von FontAwesomne müssen in die ThirdPartyNotice